docs/man_pages/project/testing/livesync.md

```diff
@@ -0,0 +1,32 @@
+livesync
+==========
+
+Usage | Synopsis
+------|-------
+General | `$ tns livesync [<Command>]`
```

Comment:
@ikoevska The platform is indeed optional and when it is not specified livesync is performed for both platforms